### PR TITLE
Intergration060121

### PR DESCRIPTION
**Pull request by tltk90**

### CODE REVIEW

File: scripts/fix-package-json.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function fix(fileName) {
 	packageJson.version = mainPackage.version;
 	packageJson.license = mainPackage.license;
 
-	if (packageJson.peerDependencies) {
+	/*if (packageJson.peerDependencies) {
```
Comment (sulimani55) [inline scripts/fix-package-json.js:14]: remove?

File: src/app/@ansyn/ansyn/package.json [hunk 2]
```diff
@@ -2,39 +2,41 @@
...
 	"version": "3.0.0",
 	"peerDependencies": {
-		"@ngrx/core": "^1.2.0",
-		"@ngrx/store": "^9.0.0",
-		"@ngrx/effects": "^9.0.0",
-		"@angular/common": "~10.0.5",
-		"@angular/core": "~10.0.5",
+		"@ngrx/store": "9.x",
+		"@ngrx/effects": "9.x",
+		"@angular/common": "10.0.0-PLACEHOLDER",
```
Comment (sulimani55) [inline src/app/@ansyn/ansyn/package.json:7]: what does placeholder mean?

File: src/app/@ansyn/imagery-cesium/tsconfig.lib.prod.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.lib.json",
   "angularCompilerOptions": {
-    "enableIvy": true
+    "enableIvy": false
```
Comment (sulimani55) [inline src/app/@ansyn/imagery-cesium/tsconfig.lib.prod.json:4]: why did you disable ivy?

Comment (tltk90) [reply src/app/@ansyn/imagery-cesium/tsconfig.lib.prod.json:4]: npm was not let you to publish package that compile with ivy,
so in production we change the ivy to false

File: scripts/fix-package-json.js (hunk 1 above)
Comment (tltk90) [reply scripts/fix-package-json.js:14]: fix

File: src/app/@ansyn/ansyn/package.json (hunk 2 above)
Comment (tltk90) [reply src/app/@ansyn/ansyn/package.json:7]: it should say to the customer to install any angular version he want but it should be equal to all.
I change it to 9 or 10.

**Review by sulimani55 — APPROVED** [review]

**Pull request merged**